src/ensembl/src/content/app/custom-download/containers/content/filter-accordion/sections/proteins/Proteins.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useEffect } from 'react';
+import { connect } from 'react-redux';
+import { RootState } from 'src/store';
+
+import ContentBuilder from 'src/content/app/custom-download/components/content-builder/ContentBuilder';
+
+import {
+  getSelectedFilters,
+  getContentState
+} from '../../state/filterAccordionSelector';
+
+import {
+  updateSelectedFilters,
+  updateContentState
+} from '../../state/filterAccordionActions';
+
+import JSONValue from 'src/shared/types/JSON';
+import allFilters from 'src/content/app/custom-download/sample-data/filters';
+import { AttributeWithContent } from 'src/content/app/custom-download/types/Attributes';
+
+type Props = StateProps & DispatchProps;
+
+const Proteins = (props: Props) => {
+  useEffect(() => {}, [props.selectedFilters]);
+
+  const content = allFilters['proteins'] as AttributeWithContent;
+  return (
+    <div>
```

Comment:
What’s the purpose of the wrapper div? ContentBuilder has its own top-level div.

Reply:
The ContentBuilder was initially returning undefined when the data was no defined. This was causing some error so I added a wrapper to get around it. The ContentBuilder was updated later on to return `null` and I missed to remove these wrappers.